src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example3.java

```diff
@@ -10,30 +10,33 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
 // xdoc section -- start
-public class MyTest {
-  public MyTest()  {          // constructor (line 1)
-      int var1 = 2;           // line 2
-      int var2 = 4;           // line 3
-      int sum = var1 + var2;  // line 4
+public class Example3 {
+
+  public Example3()  { // constructor (line 1)
+    int var1 = 2;           // line 2
+    int var2 = 4;           // line 3
+    int sum = var1 + var2;  // line 4
   } // line 5, OK, constructor is not mentioned in the tokens
 
   public void firstMethod() { // line 1
-      // comment - not counted as line
-      System.out.println("line 2");
+    // comment - not counted as line
+    System.out.println("line 2");
   } // line 3, OK, as it allows at most 4 lines
 
-  public void secondMethod() { // line 1
-      int index = 0;   // line 2
-      if (index < 5) { // line 3
-          index++;     // line 4
-      }                // line 5
+  public void secondMethod() { // violation
+    int index = 0;   // line 2
+    if (index < 5) { // line 3
+      index++;     // line 4
+    }                // line 5
   } // line 6, violation, as it is over 4 lines
```

Comment:
This is confusing. Do we really have two violations on this one method? If not, please remove any extraneous violations. Please do this for all examples.

Reply:
done